content/articles/one-way-anova-excel-testing-normality-homogeneity-using-python/index.md

```diff
@@ -0,0 +1,154 @@
+<!--more-->
+One of these methods is the analysis of variance (ANOVA). ANOVA is a set of statistical models that analyses the differences among means. Microsoft came up with statistical add-ins within Excel to solve some of these problems. Although Microsoft Excel is still rather limited. Later came tools like R-studio and Python. 
+
+With Python, the test for normality and homogeneity became easier. This tutorial will focus on creating an ANOVA table using Microsoft Excel, box plots to test the normality of a one way ANOVA, and Bartlett's rule for homogeneity with Python.
+
+### Prerequisites
+This tutorial requires readers to have proper knowledge on the following:
+- Here is a link to the [Analysis of variance (ANOVA)](https://www.investopedia.com/terms/a/anova.asp) documentation.
+- [Microsoft Excel](https://www.guru99.com/excel-tutorials.html).
+- [Python](https://www.python.org/).
+
+### Goal
+At the end of the tutorial, the reader will be able to:
+- Use Excel to create an ANOVA table.
+- Install necessary Python dependencies to create a box plot.
+- Test for normality using a box plot.
+- Test for homogeneity using Bartlett's rule.
+
+### Setting up the environment
+To get started, we'll need to install Python and some of its dependencies. [Here](https://www.python.org/downloads/) is a link to download the latest Python executable based on your operating system. 
+
+We need to install the following dependencies:
+- pandas
+- matplotlib
+- seaborn
+
+#### Installing pandas, matplotlib, seaborn
+There are several methods to install Python dependencies, the most popular being the use of package managers like pip or conda. 
+
+To install `pandas`, we'll need to:
+- Open our terminal (either our ide's terminal or command prompt).
+
+>Note: If you are using the command prompt, make sure to add the default folder to PATH.
+
+- Type in the following command:
+```bash
+python -m pip install pandas
+```
+
+Wait a moment till the necessary dependencies are installed. 
+
+Next, we'll follow the same steps for the other dependencies using the following commands respectively:
+```bash
+python -m pip install matplotlib
+```
+
+```bash
+python -m pip install seaborn
+```
+
+After installing these dependencies, we'll proceed to create an ANOVA table with Excel.
+
+### Getting started
+#### Creating an ANOVA table with Excel
+Microsoft Excels's Analysis Toolpak add-in is a game-changer in statistical Analysis. We'll show you how to create an ANOVA table containing the sum of squares, degree of freedom, mean squares, F value, P-value, F critical.
+
+To perform a one way ANOVA, implement the following step.
+- Import your data set in any preferred Excel format.
+
+![data set](/engineering-education/Solving-a-one-way-anova-with-Excel-and-testing-for-normality-and-homogeneity-using-python/data-set.jpg)
+
```

Comment:
This is not the correct link for the image. You need to confirm the correct link from your files